ArduPlane/quadplane.cpp

```diff
@@ -631,9 +631,7 @@ bool QuadPlane::setup(void)
     case AP_Motors::MOTOR_FRAME_TAILSITTER:
         break;
     default:
-        hal.console->printf("Unknown frame class %u - using QUAD\n", (unsigned)frame_class.get());
-        frame_class.set(AP_Motors::MOTOR_FRAME_QUAD);
-        setup_default_channels(4);
+        AP_BoardConfig::config_error("Unknown frame class %u", (unsigned)frame_class.get());
```

Comment:
Doesn't seem like a good idea to ignore the frame_class parameter if it's invalid, but what if there are people flying this way?